homeassistant/components/vacuum/__init__.py

```diff
@@ -292,7 +292,10 @@ def send_command(
         raise NotImplementedError()
 
     async def async_send_command(
-        self, command: str, params: dict | list | None = None, **kwargs: Any
+        self,
+        command: str,
+        params: dict[str, Any] | list[Any] | None = None,
```

Comment:
Strictly speaking, the validation is only for `dict[Any, Any]` but I think `dict[str, Any]` should be fine.